about.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Mayor's Fund for Philadelphia</title>
```

Comment:
Just a future note to have SEO friendly titles, like "About the Fund - The Mayor's Fund for Philadelphia | A nonprofit that seeks to improve the quality of lives for all Philadelphians"